endpoints/openrtb2/auction.go

```diff
@@ -1391,22 +1394,37 @@ func (deps *endpointDeps) processStoredRequests(ctx context.Context, requestJson
 					err = fmt.Errorf("imp.ext.prebid.storedrequest.id %s: Stored Imp has Invalid JSON: %s", impIds[i], Err)
 				}
 			}
-			return nil, []error{err}
+			return nil, impToStoredReq, []error{err}
 		}
 		imps[idIndices[i]] = resolvedImp
+
+		includeStoredImps, err := jsonparser.GetBoolean(resolvedImp, "ext", "prebid", "options", "echovideoattrs")
+		if err == jsonparser.KeyPathNotFoundError {
+			err = nil
+		}
```

Comment:
You can just merge these two if conditions as:
```
if err != nil && err != jsonparser.KeyPathNotFoundError {
	return nil, nil, []error{err}
}
```

Reply:
Yes, modified!